generator/dart/packages/google_cloud_common/lib/common.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+
+import 'package:http/http.dart';
+
+const _clientName = 'dart-test-client';
```

Comment:
For a future PR: there is an elaborate set of things we need to set in this header: [go/cloud-api-headers-2019](https://goto.google.com/cloud-api-headers-2019)